packages/swh-plugins.lv2/PKGBUILD

```diff
@@ -0,0 +1,39 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+_name=lv2
+pkgname=swh-plugins.lv2
+pkgver=1.0.16
+pkgrel=1
+pkgdesc="LV2 port of Steve Harris' plugins suite"
+arch=(x86_64 aarch64)
+url='http://plugin.org.uk/'
+license=(GPL3)
+groups=(lv2-plugins pro-audio)
+depends=()
+makedepends=(fftw libxslt)
+checkdepends=(kxstudio-lv2-extensions lilv lv2lint)
+optdepends=('lv2-host: for running the plugins')
+provides=(swh-plugins)
+source=("$pkgname-$pkgver.tar.gz::https://github.com/swh/$_name/archive/refs/tags/v$pkgver.tar.gz")
+sha256sums=('bc24512de6e2fb7a493226e2e01a80ba8462a318b15c3b0fd0cd914b018c3548')
+
+build() {
+  cd $_name-$pkgver
+  make
+}
+
+check() {
```

Comment:
You could insert this here:

```sh
  cd $_name-$pkgver
  for _dir in plugins/*.lv2; do
    if [ ! -e "$_dir/manifest.ttl" ]; then
      rm -rf "$_dir"
    fi
  done
```

Reply:
Alright, I wasn't sure if it's a good idea to delete arbitrary source files during `check()`.

Reply:
I added that to the `build()`, seemed more correct IMO.